cves/kernel/CVE-2016-0823.yml

```
@@ -221,8 +226,10 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
-  note:
+  name: fs
```

It might be good to write this out fully for the readability of people less familiar with OS stuff
```suggestion
  name: Filesystem
```